### PR TITLE
fix: adding logs for renderer

### PR DESCRIPTION
**Pull request by tonysnowboardunderthebridge**
Hi @marcopiraccini 

adding logs and some changes related to https://github.com/platformatic/meraki/issues/251

### CODE REVIEW

File: src/renderer/src/components/steps/ConfigureApplication.jsx [hunk 1]
```diff
@@ -15,8 +15,11 @@ import Title from '~/components/ui/Title'
 import '~/components/component.animation.css'
 import Forms from '@platformatic/ui-components/src/components/forms'
 import { TYPESCRIPT, JAVASCRIPT } from '~/ui-constants'
+import log from 'electron-log/renderer'
 
 const ConfigureApplication = React.forwardRef(({ onNext, onBack }, ref) => {
+  console.log = log.log
+  Object.assign(console, log.functions)
```
Comment (marcopiraccini) [inline src/renderer/src/components/steps/ConfigureApplication.jsx:22]: Not sure it's a good idea to monkey-patch console. log.

**Review by marcopiraccini — CHANGES_REQUESTED** [review]
I think we should find another way of doing this.

File: src/renderer/src/components/steps/ConfigureApplication.jsx (hunk 1 above)
Comment (tonysnowboardunderthebridge) [reply src/renderer/src/components/steps/ConfigureApplication.jsx:22]: I taken from the https://github.com/megahertz/electron-log?tab=readme-ov-file#renderer-process

Comment (tonysnowboardunderthebridge) [reply src/renderer/src/components/steps/ConfigureApplication.jsx:22]: Schermata 2024-02-21 alle 16 56 01

Comment (tonysnowboardunderthebridge) [reply src/renderer/src/components/steps/ConfigureApplication.jsx]: Done!

**Review by marcopiraccini — APPROVED** [review]
lgtm

**Pull request merged**